docs/maui/converters/math-expression-converter.md

```diff
@@ -0,0 +1,124 @@
+---
+title: MathExpressionConverter - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
+description: "The MathExpressionConverter is a converter that allows users to perform various math operations."
+ms.date: 03/16/2022
+---
+
+# MathExpressionConverter
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `MathExpressionConverter` is a converter that allows users to perform various math operations.
```

Comment:
Perhaps we should make it clear this only works with a single binding value and refer the reader to the multi option:

```suggestion
The `MathExpressionConverter` is a converter that allows users to perform various math operations. This works with a single binding value, if you require multiple values through a `MultiBinding` then see [`MultiMathExpressionConverter`](multi-math-expression-converter.md)
```